File: BackEnd/MaladiesCronique/APIMaladiesCronique/Models/PatientViewModel/Patient.cs
namespace APIMaladiesCronique.Models.PatientViewModel
{
    public class Patient
    {
        public Guid Id { get; set; }
        public required string Nom { get; set; }
        public required string  Prenom { get; set; }
        public required string  Cin { get; set; }
        public DateOnly? DateNaissance { get; set; }
        public string? Adresse { get; set; }
        public string? NumeroOrder { get; set; }
        public required string Sexe { get; set; }
        public DateOnly? DateDebutMaladie { get; set; }
        public string? NumeroTelephone { get; set; }
        public int TypeDiabete { get; set; }
        
    }
}

File: FrontEnd/MaladiesCronique/src/app/apis/patient.ts
export default interface Patient {
    Id: string,
    Nom: string,
    Prenom: string,
    Cin: string,
    DateNaissance: Date,
    Adresse: string,
    NumeroOrder: string,
    Sexe: string,
    DateDebutMaladie: Date,
    NumeroTelephone: string,
    TypeDiabete: Int16Array
}